src/cmd/dut.rs

```diff
@@ -212,9 +213,23 @@ fn run_dut_monitor(args: &ArgsDutMonitor) -> Result<()> {
     let mut targets: Vec<MonitoredDut> = Vec::new();
     let mut port = 4022;
 
-    for dut in &args.duts {
-        targets.push(MonitoredDut::new(dut, port)?);
+    for raw_dut in &args.duts {
+        let ports: Vec<&str> = raw_dut.split(',').collect();
```

Comment:
instead of a direct string cant our command line parser handle this?

Reply:
Hmm, what syntax do you expect for this feature?
Command line parser expects only `--option option_arg` or  positional arguments which separated by spaces. So I think it is not possible to specify *which* ip&port is forwarded from which DUT.

Reply:
i guess argh doesnt support it